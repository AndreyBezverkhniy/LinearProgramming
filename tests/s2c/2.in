3
2
0 -3 0.5
-10.1 20.2 -30.3 40.4
10 20 30 40

n # оригинальных переменных
m # уравнений
c1 .. cn
...
a{Bi}1 ... a{Bi}n <= b{Bi}
...

