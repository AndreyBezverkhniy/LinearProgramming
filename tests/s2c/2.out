z = 0 + (0)*x0 + (-3)*x1 + (0.5)*x2 + (0)*x3 + (0)*x4
x3 = 40.4 - (-10.1)*x0 - (20.2)*x1 - (-30.3)*x2 - (0)*x3 - (0)*x4
x4 = 40 - (10)*x0 - (20)*x1 - (30)*x2 - (0)*x3 - (0)*x4
